es6-shim.js

```diff
@@ -1438,7 +1438,7 @@
       if (number === 0) {
         return 32;
       }
-      return 32 - (number).toString(2).length;
+      return 31 - Math.floor(Math.log(number) * Math.LOG2E + (1 / 2 / 4294967296));
```

Comment:
I think, it is not needed to put it outside. And it is hard for me to think of name...